server/test/ping.spec.js

```diff
@@ -1,22 +1,18 @@
+require("dotenv").config({ path: __dirname + "/../sample.env" });
 const chai = require("chai");
 const chaiHttp = require("chai-http");
-const app = require("../app.js");
+const { app } = require("../app.js");
 
 chai.should();
 chai.use(chaiHttp);
 
-describe("/POST ping", () => {
-  it("it should return 200 and message", (done) => {
-    chai
-      .request(app)
-      .post(`/ping/`)
-      .send({ message: "hello" })
-      .end((err, res) => {
-        res.should.have.status(200);
-        res.body.should.have
-          .property("response")
-          .eql("Server is running. Message received: hello");
-        done();
-      });
+describe("Ping test with logout controller", () => {
+  it("should return 200 and message", async () => {
+    const testingMessage = { message: "Ping test with logout controller" };
+    const res = await chai.request.agent(app).post("/auth/logout").send(testingMessage);
+    res.should.have.status(200);
+    res.should.have
+      .property("text")
+      .eql(`Server is running. Message received: ${testingMessage.message}`);
   });
 });
```

Comment:
Already reviewed in the previous PR. Consider using `git rebase` if you don't want this kind of change to show as new changes in the new branch you checked out to. After making a commit in one branch, if you checkout to a new feature branch, you don't want the changes in the old branch to still show up as new changes in the new branch since you've already made a PR on that, you can just do `<in the new branch> git rebase <old_branch>`.